transport/tchannel/inbound_test.go

```diff
@@ -141,5 +139,58 @@ func TestInboundSubServices(t *testing.T) {
 	}
 
 	require.NoError(t, i.Stop())
-	require.NoError(t, itransport.Stop())
+	require.NoError(t, it.Stop())
+	require.NoError(t, o.Stop())
+}
+
+func TestArbitraryInboundServiceOutboundCallerName(t *testing.T) {
+	it, err := NewTransport(ServiceName("service"))
+	require.NoError(t, err)
+	i := it.NewInbound()
+	i.SetRouter(transporttest.EchoRouter{})
+	require.NoError(t, i.Start(), "failed to start inbound")
+	require.NoError(t, it.Start(), "failed to start inbound transport")
```

Comment:
you start inbound then inbound transport. below you start outbound transport then outbound. This sounds confusing to me, but it might be correct. Just pointing that out.